corehq/apps/domain/models.py

```diff
@@ -48,6 +49,19 @@
         lang_lookup[lang['two']] = lang['names'][0]
 
 
+class _StrictQuickCache(QuickCache):
```

Comment:
I like this, seems almost generalizable, but probably good idea to keep that out of scope for now.

One question—it seems like the original function would cache the response even in strict mode, and I think that's probably the right behavior. Does this also cache the response when strict=True?